README.md

```diff
@@ -145,23 +145,23 @@ Note that if middleware is not used in the Next.js app, the `middleware-function
 
 #### WORKAROUND: `public/` static files served out by server function (AWS specific)
 
-Recall in the [S3 bucket](#s3-bucket) section, files in your app's `public/` folder are staitc, and are uploaded to the S3 bucket. Ideally, requests to these files should be handled by the S3 bucket. For example:
+Recall in the [S3 bucket](#s3-bucket) section, files in your app's `public/` folder are static, and are uploaded to the S3 bucket. Ideally, requests to these files should be handled by the S3 bucket. For example:
 
 ```
 https://my-nextjs-app.com/favicon.ico
 ```
 
 This requires the CloudFront distribution to have the behavior `/favicon.ico`, and set the S3 bucket as the origin. However, CloudFront has a [default limit of 25 behaviors per distribution](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/cloudfront-limits.html#limits-web-distributions). It is not a scalable solution to create 1 behavior per file.
 
-To workaround the issue, requests to `public/` files are handled by the cache all behavior `/*`. The behavior sends the request to the server function first. And if the server fails to handle the request, it will fallback to the S3 bucket.
+To work around the issue, requests to `public/` files are handled by the cache all behavior `/*`. The behavior sends the request to the server function first. And if the server fails to handle the request, it will fallback to the S3 bucket.
```

Comment:
```suggestion
To workaround the issue, requests to `public/` files are handled by the cache all behavior `/*`. The behavior sends the request to the server function first. And if the server fails to handle the request, it will fallback to the S3 bucket.
```

Reply:
`workaround` is a noun. I think we want `work around` as a verb?